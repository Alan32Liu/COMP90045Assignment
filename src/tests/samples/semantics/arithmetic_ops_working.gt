proc main()
begin
    write 1 + 1;
    write 2 * 1;
    write 4 / 2;
    write 6 - 4;
    write "\n";
    write 1.0 + 1.0;
    write 2.0 * 1.0;
    write 4.0 / 2.0;
    write 6.0 - 4.0;
    write "\n";
end
