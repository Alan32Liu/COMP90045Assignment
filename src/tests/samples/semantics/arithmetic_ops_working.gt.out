2222
2.0000002.0000002.0000002.000000
